env.d/development/dev-ssl

```diff
@@ -4,5 +4,6 @@ AUTHENTICATION_BACKEND=richie.apps.courses.lms.edx.TokenEdXLMSBackend
 
 # LMS Backend
 EDX_BACKEND=richie.apps.courses.lms.edx.TokenEdXLMSBackend
+EDX_COURSE_REGEX=^.*/courses/(?P<course_id>.*)/info$
```

Comment:
Since you said /info redirects to /course :man_shrugging: :
```suggestion
EDX_COURSE_REGEX=^.*/courses/(?P<course_id>.*)/course$
```

Reply:
Yes, I will also update `create_demo_site.py` !